content/homepage/accommodation.md

```diff
@@ -5,13 +5,10 @@ header_menu_title: "Accommodation"
 header_menu: true
 ---
 
-To help your search for accommodation in Hamburg, you can find a list of **[hotel options](https://owncloud.gwdg.de/index.php/s/c4HlclYFUgFGf9Q)**.
-For easier orientation, the list of hotels can also be viewed on **[Google Maps](https://maps.app.goo.gl/NJGnAixVdawyriht8).**
+To help your search for accommodation in Hamburg, you can find a list of [**hotel options**]({{% ref hotellist %}}).
+For easier orientation, the list of hotels can also be viewed on **[Google Maps](https://www.google.de/maps/search/Hotels/@53.5668599,9.9604056,15.66z?entry=ttu&g_ep=EgoyMDI1MDIxMC4wIKXMDSoASAFQAw%3D%3D)**.
 
-Alternatively, you can book hotel rooms on the following websites:
+**Please Note!** The weekend before the dates of our hackathon Hamburg celebrate its [836th port anniversary](https://www.hamburg.com/visitors/port-anniversary). Hotel may stay above their regular leve. **To avoid overly high prices, book as soon as possible with the option for flexible cancellation!**
```

Comment:
```suggestion
**Please Note!** The weekend before the dates of our hackathon Hamburg celebrate its [836th port anniversary](https://www.hamburg.com/visitors/port-anniversary). Hotel prices may stay above their regular level. **To avoid overly high prices, book as soon as possible with the option for flexible cancellation!**
```